src/sentry/data_export/tasks.py

```diff
@@ -84,14 +84,8 @@ def assemble_download(
             return
 
         with sentry_sdk.configure_scope() as scope:
-            if data_export.user:
-                user = {}
-                if data_export.user.id:
-                    user["id"] = data_export.user.id
-                if data_export.user.username:
-                    user["username"] = data_export.user.username
-                if data_export.user.email:
-                    user["email"] = data_export.user.email
+            if data_export.user_id:
+                user = dict(id=data_export.user_id)
```

Comment:
Trimming down to only user_id seems reasonable as getting the user will use another RPC call.